website/docs/components/form/masked-input/partials/guidelines/guidelines.md

```diff
@@ -68,6 +68,12 @@ For shorter, simpler forms, indicate optional fields instead.
   <F.Label>Label</F.Label>
 </Hds::Form::MaskedInput::Field>
 
+## Copyable masked input 
+
+Setting the `hasCopyButton` property to `true` will display a [Copy Button](/components/copy/copy-button) adjacent to the Masked Input, allowing users to copy the value in the Masked Input to their clipboard. Refer to the [Copy Button](/components/copy/copy-button) usage guidelines for more details around copying content. 
+
+![Masked Input with a Copy Button](/assets/components/form/masked-input/masked-input-has-copy-button.png)
```

Comment:
Using the actual component to show the functionality may be more impactful because they can see it in action. I realize it's not available in code yet, but the docs can't be merged until they're available anyway.

Once ready, you should be able to use the following code: 

```suggestion
<Doc::Layout @spacing="16px">
  <Hds::Form::MaskedInput::Field @value="0362df4996ca864b4df9e42cba0d6d" @width="300px" @hasCopyButton={{true}} as |F|>
    <F.Label>Label</F.Label>
  </Hds::Form::MaskedInput::Field>
  <Hds::Form::MaskedInput::Field @value="0362df4996ca864b4df9e42cba0d6d" @isMultiline={{true}} @width="300px" @hasCopyButton={{true}} as |F|>
    <F.Label>Label</F.Label>
  </Hds::Form::MaskedInput::Field>
</Doc::Layout>
```

@alex-ju can you double-check that this code would be accurate?

Reply:
yes, that's how it should look like. Haven't got the chance to work on the code yet, but I'll try and make it a priority to avoid this PR stalling for too long.

Reply:
Thanks @alex-ju!